index.html

```diff
@@ -141,7 +142,7 @@ <h3 class="article__title">
           <div class="article">
             <img
               class="article__thumbnail"
-              src="edinburgh-damascus_thumbnail.png"
+              src="../images/edinburgh-damascus_thumbnail.png"
```

Comment:
These images aren't displaying for me, can you fix them?

Reply:
I updated and wrote alt's as forgot before :)
Let me know if the image is working.

One question, please. Do you have a tool where you can check the website without downloading it to your computer?

Reply:
Great that you added the `alt` attributes! The images aren't displaying for me, because the file path is `./images`, instead of `../images`, because the index.html is in the same directory as the `images` folder